server/routes/auth.ts

```diff
@@ -273,7 +273,7 @@ app.use(passport.session());
 export let authRoutes = express.Router();
 
 function getExternalPort(request: express.Request): number {
-	let host = request.headers.host!;
+	let host = request.headers.host as string;
```

Comment:
Does this convert `null` / `undefined` into a string too? Maybe `as string | undefined` ?